### ISSUE TITLE
Contrast issues on clicking block name

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#2255]
The contrast of the grey and blue on clicking a block to insert makes it hard to read. Whilst you aren't reading so much, it should be at least improved as looks like something goes wrong and I suspect doesn't pass an accessibility check.

![2017-08-07 09 23 20](https://user-images.githubusercontent.com/253067/29018267-623a369e-7b52-11e7-827b-1228d84981a3.gif)

**Comment by afercia** [comment]
See also #1355 for the :focus part.

**Comment by mtias** [comment]
Is this still an issue after the inserter changes?

**Comment by afercia** [comment]
I think this can be closed now? @karmatosed
